packages/apps/tasks/src/proto/schema.proto

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright 2023 DXOS.org
+//
```

Comment:
Do I need these comments at the top of every file?

Reply:
Within the monorepo the linted will enforce this, yes